packages/react-server-gulp-route-loader/package.json

```diff
@@ -0,0 +1,18 @@
+{
+  "name": "react-server-gulp-route-loader",
+  "version": "0.0.1",
+  "description": "A gulp plugin to handle lazy-loading of react-server Page objects to support code-splitting with webpack",
+  "main": "index.js",
```

Comment:
Wait... it looks like it's called `main.js` above.  Does this work?

FWIW - I think `index.js` is the more common convention.